src/DssSpell.sol

```diff
@@ -301,8 +301,9 @@ contract DssSpellAction is DssAction {
             // 80M * 1.04^5 * 1.00 as a WAD
             RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
                 "RWA012-A",
-                 97_332_232 * WAD
+                 97_332_233 * WAD
```

Comment:
Yeah I think sometimes the tests fail unless you add `1` to `line` so that might explain it.

Noting here that the correct value is 97,332,232 by calculating sum of old `val` values, but according to the test coverage the counterparty can still only mint exactly 80M DAI before interest / SF is accounted for (i.e. net of interest over the 60 month term).

Worst case, the counterparty is able to mint an excess of 1 (unbacked) DAI which can be settled at time of deal closure and is immaterial relative to the other amounts involved in the deal.

OK with this rounding change on my end given these factors